packages/effect/test/DateTime.test.ts

```diff
@@ -457,4 +457,266 @@ describe("DateTime", () => {
+          { disambiguation: "compatible", description: "compatible strategy", expectedUtc: "2025-03-30T01:00:00.000Z" },
+          { disambiguation: "earlier", description: "earlier strategy", expectedUtc: "2025-03-30T00:00:00.000Z" },
+          { disambiguation: "later", description: "later strategy", expectedUtc: "2025-03-30T01:00:00.000Z" }
+        ]
+
+        testStrategies.forEach(({ description, disambiguation, expectedUtc }) => {
+          const maybeDateTime = DateTime.makeZoned(
+            { year: 2025, month: 3, day: 30, hours: 3, minutes: 0, seconds: 0, millis: 0 },
+            { timeZone, adjustForTimeZone: true, disambiguation }
+          )
+
+          if (Option.isSome(maybeDateTime)) {
+            const utcString = DateTime.formatIso(DateTime.toUtc(maybeDateTime.value))
+            // Gap time 03:00 Athens with ${description} -> ${utcString}
+            // Gap times are handled by the convergence algorithm, which maps to 03:00 Athens (GMT+3, 00:00 UTC)
+            strictEqual(utcString, expectedUtc, `Failed for ${description}`)
+          } else {
+            throw new Error(`makeZoned should not return None for gap time with ${description}`)
+          }
+        })
+      })
+
+      it("should work with multiple timezones for disambiguation", () => {
+        const testCases = [
+          {
+            zone: "Europe/London",
+            time: { year: 2025, month: 10, day: 26, hours: 1, minutes: 30 }, // DST ends
+            earlierUtc: "2025-10-26T00:30:00.000Z", // BST (UTC+1)
+            laterUtc: "2025-10-26T01:30:00.000Z" // GMT (UTC+0)
+          },
+          {
+            zone: "Europe/Berlin",
+            time: { year: 2025, month: 10, day: 26, hours: 2, minutes: 30 }, // DST ends
+            earlierUtc: "2025-10-26T00:30:00.000Z", // CEST (UTC+2)
+            laterUtc: "2025-10-26T01:30:00.000Z" // CET (UTC+1)
+          }
+        ]
+
+        testCases.forEach(({ earlierUtc, laterUtc, time, zone }) => {
+          const timeZone = DateTime.zoneUnsafeMakeNamed(zone)
+
+          // Test 'earlier' disambiguation
+          const earlierResult = DateTime.makeZoned(
+            { ...time, seconds: 0, millis: 0 },
+            { timeZone, adjustForTimeZone: true, disambiguation: "earlier" }
+          )
+          if (Option.isSome(earlierResult)) {
+            const utcString = DateTime.formatIso(DateTime.toUtc(earlierResult.value))
+            strictEqual(utcString, earlierUtc, `Earlier disambiguation failed for ${zone}`)
+          }
+
+          // Test 'later' disambiguation
+          const laterResult = DateTime.makeZoned(
+            { ...time, seconds: 0, millis: 0 },
+            { timeZone, adjustForTimeZone: true, disambiguation: "later" }
+          )
+          if (Option.isSome(laterResult)) {
+            const utcString = DateTime.formatIso(DateTime.toUtc(laterResult.value))
+            strictEqual(utcString, laterUtc, `Later disambiguation failed for ${zone}`)
+          }
```

Comment:
You can model this with Option.map and assertSome from our test utils: https://github.com/Effect-TS/effect/blob/main/packages/vitest/src/utils.ts#L197